api/src/main/java/io/druid/data/input/impl/ParseSpec.java

```diff
@@ -22,12 +22,12 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonSubTypes;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
-
 import io.druid.java.util.common.parsers.Parser;
 
 import java.util.List;
 
 /**
+ * Not an {@code ExtensionPoint} since extension parsers are meant to extend {@code InputRowParser}.
```

Comment:
given that InputRowParser is an extension point, this needs to be an extension point or else how would they implement InputRowParser.getParseSpec() ?